site/content/docs/cli/command-line-reference.md

```diff
@@ -183,7 +183,7 @@ Pull all environment variables to the `.env` file and overwrite it if it already
 
 ### `npx checkly env ls`
 
-List global environment variables. This command does not list environment variables on group or check level.
+List global environment variables. This command does not list environment variables on group or check level. For secrets, only the key will be shown, not the value.
```

Comment:
```suggestion
List global variables. This command does not list  variables on group or check level. For secrets, only the key name will be shown, not its value.
```

Reply:
Isn't the key the 'name' in a key-value pair? This reads like salsa sauce to me 😄

Reply:
It is, but the key in a very general way can refer to either. Given that we mention the "value", I thought it made sense to also specify the "name".